src/app/personside/visittkort-v2/visittkort-utils.ts

```diff
@@ -73,8 +71,7 @@ export function hentAlderEllerDodRelasjon(relasjon: ForelderBarnRelasjon): strin
 }
 
 export function relasjonErDod(relasjon: ForelderBarnRelasjon): boolean {
-    const personstatus = relasjon.personstatus.firstOrNull();
-    return !!personstatus && personstatus.kode === PersonStatus.DOD;
+    return !relasjon.dodsdato.isEmpty();
```

Comment:
Tenkte vi kunne endre på `erDod` og `relasjonErDod` til å bare ta inn `dodsdato` i stedet for `person` og `relasjon`, for da trenger vi kun å beholde `erDod` og bruke den overalt😄 

Og så kan vi bruke `.isNotEmpty()` i stedet for "not" `.isEmpty()` 🙌🏼